admin_guide/master_node_configuration.adoc

```diff
@@ -10,44 +10,116 @@
-The `openshift start` command accepts flags that indicate that it should simply write the config file that it would have used and terminate.  This is useful for getting a starting point for the config.  You can do this by running 
+The configuration files for masters and nodes are fully specifying with no
+defaulting. This means that any empty value means that you want to start up with
+an empty value for that parameter. It makes it easy to reason about exactly what
+your configuration is, but it also means that it can be difficult to remember
+all of the options to specify. To make this easier, the configuration files can
+be created with the `--write-config` flag and then used with the `--config`
+flag.
 
-* `openshift start --write-config=openshift.local.config`
-* `openshift start master --write-config=openshift.local.config/master`
-* `osadm create-node-config --node-dir=openshift.local.config/node-myhost --node=myhost --hostnames=myhost.local,10.0.0.1`
+== Creating New Configuration Files
+For masters, the `openshift start` command accepts flags that indicate that it
+should simply write the configuration file that it would have used, then
+terminate. For nodes, a configuration file can be written using the `osadm
+create-node-config` command. Creating new configuration files is useful for
+getting a starting point for defining your configuration.
 
-== Use the config files
-Once you have modified the config files to your liking, you can make use of them by specifying them as an argument.  Keep in mind that if you specify a config file, *none of the other flags you pass in will be respected*.  You can run them like: 
+The following commands write the relevant launch configuration file(s),
+certificate files, and any other necessary files to the specified
+`--write-config` or `--node-dir` directory.
 
-* `openshift start --master-config=openshift.local.config/master/master-config.yaml --node-config=openshift.local.config/node-<node-name>/node-config.yaml`
-* `openshift start master --config=openshift.local.config/master/master-config.yaml`
-* `openshift start node --config=openshift.local.config/node-myhost/node-config.yaml`
+To create a configuration file for an all-in-one server (a master and a node on
+the same host), run:
 
-== Stub for discussion of generating the config for a new node
+****
+`$ openshift start --write-config=openshift.local.config`
+****
 
+To create a configuration file for a master:
 
-== Example master.yaml
-This is an example at a point in time.  You should run `--write-config` to generate a file of your own.
+****
+`$ openshift start master --write-config=openshift.local.config/master`
+****
 
-[source]
----
+To create a configuration file for a node:
+
+****
+`$ osadm create-node-config
+--node-dir=openshift.local.config/node-_<node_hostname>_
+--node=_<node_hostname>_ --hostnames=_<hostname>_,_<ip_address>_`
+****
+
+For the `--hostname` flag in the above command, use a comma-delimited list of
```

Comment:
is it --hostname or --hostnames? You have hostnames in your example, but refer to it by hostname.